test/e2e/framework/podlogs/podlogs.go

```diff
@@ -133,7 +133,11 @@ func CopyAllLogs(ctx context.Context, cs clientset.Interface, ns string, to LogO
 					var prefix string
 					if to.LogWriter != nil {
 						out = to.LogWriter
-						prefix = name + ": "
+						nodeName := pod.Spec.NodeName
+						if len(nodeName) > 10 {
+							nodeName = nodeName[0:5] + ".." + nodeName[len(nodeName)-5:]
```

Comment:
suggesting to account for `..` in the clamped length.

e.g. a 5..3:
```
nodeName = nodeName[0:5] + ".." + nodeName[len(nodeName)-3:]
```
or a `4..4`.
```
nodeName = nodeName[0:4] + ".." + nodeName[len(nodeName)-4:]
```

Reply:
if `5..5` is desired the check can be `> 12`.

Reply:
`nodeName = nodeName[0:4] + ".." + nodeName[len(nodeName)-4:]` seems good idea because I guess node names tend to have some specific chars at the end.

Reply:
I think I wanted to have 10 significant characters originally, but don't have a strong preference. As the math looks odd as it was and there may be merit to having the same length in all cases (think mixing output with truncated and untruncated node names), I changed it to 4+2+4 and a true maximum length of 10.